openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -227,6 +228,7 @@ def _submit_deadline_post_job(self, instance, job, instances):
         environment["AVALON_APP_NAME"] = os.environ.get("AVALON_APP_NAME")
         environment["OPENPYPE_LOG_NO_COLORS"] = "1"
         environment["OPENPYPE_USERNAME"] = instance.context.data["user"]
+        environment["OPENPYPE_SG_USER"] = os.getenv("OPENPYPE_SG_USER")
```

Comment:
This seems odd. We're explicitly adding `OPENPYPE_SG_USER` but also have it added to the `enviro_filter` to it gets filtered out below?

Reply:
Fixed